src/modules/0.0.21/iasql_functions/rpcs/iasql_preview_apply.ts

```diff
@@ -0,0 +1,36 @@
+import { snakeCase } from 'typeorm/util/StringUtils';
```

Comment:
Same general comments here, and if you find that you're repeating a lot of cruft in here, it may make sense to either bake it into `RpcBase` or provide some `super.method(...)` functions (like that `formattedRes` could become `return super.snakeCaseRecords(applyRes);`)

Reply:
💯